### ISSUE TITLE
New Ontology Request: terms4FAIRskills (t4fs)

### ISSUE THREAD

**Issue by allysonlister** [OBOFoundry/OBOFoundry.github.io#1520]
Use this form to register a new ontology with the OBO Foundry. Please read the instructions provided here:
http://obofoundry.org/docs/NewOntologyRegistrationInstructions.html

## Ontology title

terms4FAIRskills

## Requested ID space

t4fs (ID style T4FS_0000001)

## Ontology location
https://github.com/terms4fairskills/FAIRterminology (general repository)
https://github.com/terms4fairskills/FAIRterminology/blob/master/development/t4fs.owl (OWL file - release will be built with appropriate version IRI once OBO Foundry submission process is complete.)

## Contact person
Name: Allyson Lister, Laura Molloy, Peter McQuilton, Yann Le Franc
Email address: allyson.lister@oerc.ox.ac.uk, laura@codata.org, pete_mcq@protonmail.com, ylefranc@esciencefactory.com
GitHub username: allysonlister, lauramolloy, Drosophilic, ylefranc

## Issue tracker
https://github.com/terms4fairskills/FAIRterminology/issues

## Ontology license

- [ ] CC0 (public domain)
- [X] CC-BY (version 3 or later)
- [ ] Other: *please specify*

## Available ontology formats
OWL serialized as RDF/XML

## What domain is the ontology intended to cover?
terms4FAIRskills is a terminology that describes the competencies, skills and knowledge associated with making and keeping data FAIR. This terminology was created to apply to a variety of use cases, including:

- To assist with the creation and assessment of stewardship curricula;
- To facilitate the annotation, discovery and evaluation of FAIR-enabling materials (e.g. training) and resources;
- To enable the formalisation of job descriptions and CVs with recognised, structured competencies.

The terminology is of use to trainers who teach FAIR data skills, researchers who wish to identify skill gaps in their teams and managers who need to recruit individuals to relevant roles.

## Related OBO Foundry ontologies
None

## Intended use/related projects
We are working with a number of projects and use cases within EOSC-Life, the ELIXIR Training Platform and TeSS tool, FAIRassist.org, and FAIRsFAIR. Five hackathons have already been completed, where our community has annotated these training materials with T4FS terms. More information on collaborators, use cases and related projects can be found at https://terms4fairskills.github.io/

## Data source
Training materials such as slides and webinars from the community have informed the creation of the model and the ontology itself. Raw outputs from the Semaphora annotation tool from recent hackathons have informed the model, and can be found at http://t4fs.esciencedatafactory.com/. 

## Metadata

Please fill in the following metadata record which will be used by the OBO Foundry website. Note that the values shown are just examples, for example `yourfourletterid` could be something like `aism`, `cohm`, `mondo` (it does not have to be four letters). `your_domain_like_for_example_anatomy` could be simply `anatomy`, and the license should be whatever your actual license is. An example can be found [here](https://github.com/OBOFoundry/OBOFoundry.github.io/edit/master/ontology/amphx.md), but you really only need to fill in the metadata mentioned here.

```
id: t4fs
title: terms4FAIRskills
contact:
  email: pete_mcq@protonmail.com
  label: Peter McQuilton
description: A terminology for the skills necessary to make data FAIR and to keep it FAIR.
domain: skills for enabling FAIR data, applicable across all research areas including the life sciences
homepage: https://github.com/terms4fairskills/FAIRterminology
products:
  - id: t4fs.owl
dependencies:
tracker: https://github.com/terms4fairskills/FAIRterminology/issues
license:
  url: http://creativecommons.org/licenses/by/4.0/
  label: CC-BY 4
usages:
  - user: http://t4fs.esciencedatafactory.com/
    description: Semaphora integrates terms4FAIRskills, allowing users to annotate training materials with the ontology.
  - user: https://fairassist.org
description: FAIRassist is designed to offer personalised guidance to all stakeholders to enable the discovery of standards and repositories in FAIRsharing, which should be used to make data FAIR, as well as to signpost FAIR assessment resources.
  - user: https://www.fairsfair.eu/
description: FAIRsFAIR Competence Centre (project wp6) will provide a platform for training materials resulting from project training activities, annotated with the terms4FAIRskills terminology for simple finding and comparison. 
```

**Comment by alanruttenberg** [comment]
Looks like nice work. Were you thinking of aligning to BFO / other
ontologies? Some observations: There are many BFO:processes, a number of
terms that are subs or possibly equivalent to OBI:data transformation and
IAO:Information content entity. Capabilities are usually considered
realizable entities - there have been some efforts to formalize them by
Barry and friends.  The ISO documents would be considered instances of ICEs.

Best,
Alan

**Comment by allysonlister** [comment]
Hi Alan,

It's lovely to hear from you, I hope you're well in these strange times. Glad you like t4fs, thanks!

We have been thinking about aligning to some of these ontologies, but haven't done it yet for reasons I'll explain briefly, as we would appreciate your (and the rest of the foundry's) thoughts on the matter. I'll also tag my colleagues @lauramolloy, @Drosophilic, and @ylefranc in case they would like to comment.

Our use cases to date have mainly centered around iterative development of the ontology and its underlying model through the use of a number of hackathons. These hackathons were attended by our community of users, who would then search through the t4fs terms and tag digital objects (mainly training material) with those terms. If we incorporated a ULO such as BFO, we run the risk of exposing its high-level terms to the users via the annotation application. Having them see _information content entity_ in the list of available terms could be confusing.

What have other foundry ontologies done to address this? It would be great to hear about some practical experience from some of the other members.

Internally in FAIRsharing, we have an application ontology for our hierarchy of knowledge domains that draws from a large number of reference ontologies, mostly from the OBO Foundry. To hide the ULO terms from our users, we have created an annotation property that our site recognizes and uses as a filter. However, this puts the burden of filtering on the applications that use the ontology, which is not a problem for an ontology that was created for one main use case (as the FAIRsharing application ontology has been). However, it seems like a bigger deal to ask any potential tool builder that wishes to use our ontology to do the same. 

These have been our thoughts, and why the ontology currently contains terms tightly scoped to our domain. 

Thanks!

**Comment by matentzn** [comment]
Hey @allysonlister 

Thank you very much for your submission!

Before we start with our review, we are running a few basic automatic checks:

https://obofoundry.github.io/obo-nor.github.io/dashboard/index.html

Are you familiar with the OBO dashboard? Would you mind looking through the problems and see whether you can solve them before we continue?

Let us know if you have problems!

**Comment by alanruttenberg** [comment]
Hey Allyson!  Strange times indeed.

There are a couple of issues that are separable.

1) Terms for which there is an existing OBO term that means the same thing,
like some data transformation terms. In that case there's no impact to
users' view of the thing if a different IRI is used.
2) The subclass hierarchy which, as you say, needs to be filtered in a user
interface.

I don't see any reason not to fix (1).

For (2) your internal version with the annotation property to indicate
terms to be exposed in an interface is how I would expect it to be done,
and what is what I have seen in other applications. My gut would be to have
the primary ontology be the one you use internally and have the version
used for tagging be a slim/variant. Same ontology IRI, different version
IRI. That way the ontology is well integrated into OBO and applications
that want to *use* the annotations once they are made have access to a
richer version.

Best,
Alan

**Comment by allysonlister** [comment]
Thanks Alan. Apologies for the late reply, I've been on annual leave. I'll have a chat with the dev group and get back to you with a more thorough reply soon.

Best wishes,
Ally

**Comment by allysonlister** [comment]
This is just to say (generally) that I'm busy over the next few weeks with a new release of FAIRsharing, and so if there is any pause on our replies here, that's the reason. You may see one of the others reply, however. 

Thanks again!

**Comment by zhengj2007** [comment]
Discussed on OBO operations 2021-6-29 call.

@allysonlister Could you please check the dashboard report and fix the errors?
https://obofoundry.github.io/obo-nor.github.io/dashboard/t4fs/dashboard.html

Thanks!

**Comment by allysonlister** [comment]
Thank you - yes I will! We're really busy at the moment, and I'm likely to have more time for it in a week or two. Thanks!

**Comment by allysonlister** [comment]
Good morning everyone! I'm making the updates (apologies, annual leave and the impending new release of FAIRsharing had somewhat taken over the summer :-) ) and just have a few questions.

Our dashboard errors and related questions are these:

1. Registry licence. The dashboard reported a warning as follows
![registry-licence](https://user-images.githubusercontent.com/143586/134144017-db8d5ccf-2c61-4585-a051-6bb6cd1961ad.png)
Our dcterms:license field is:
`<terms:license>https://creativecommons.org/licenses/by/4.0/</terms:license>`
I am unclear as to how this fails the dashboard test? Looking at http://obofoundry.org/principles/fp-001-open.html the example code for OWL is 
`<dcterms:license rdf:resource="http://creativecommons.org/licenses/by/4.0/"/>`
As far as I can tell, there are two differences: the URL in the example is listed within an rdf:resource, and the fact that we have used https in our URL. Which of these things are causing the problem? Is our usage of the dcterms:licence incorrect?
2. Missing Version IRI. I had deliberately not made a version IRI as I was trying to hold off on an "official" release until we had a (hopefully positive :-) ) answer from you guys. The main reason for this is that http://obofoundry.org/principles/fp-004-versioning.html asks for a versionIRI in the style of
`<owl:versionIRI rdf:resource="http://purl.obolibrary.org/obo/obi/2014-12-03/obi.owl"/>` 
and the style "purl.obolibrary.org" is not a URL that we can rightly use yet. Would
`<owl:versionIRI rdf:resource="https://github.com/terms4fairskills/FAIRterminology/releases/2021-05-25/t4fs.owl"/>`
work in the meantime?

We have updated the versionInfo in this branch (https://github.com/terms4fairskills/FAIRterminology/blob/obo-foundry-updates/development/t4fs.owl), and I'll make the additional updates once I hear back from you about the best way to proceed.

Thanks very much!

**Comment by matentzn** [comment]
@allysonlister thank you for looking into this. Actually the license thing was a red herring. Just a typo of the "label" of the license, which should have been just `CC-BY`. Here is the corrected metadata record.

```
- id: t4fs
      title: terms4FAIRskills
      contact:
        email: pete_mcq@protonmail.com
        label: Peter McQuilton
      description: A terminology for the skills necessary to make data FAIR and to keep it FAIR.
      domain: skills for enabling FAIR data, applicable across all research areas including the life sciences
      homepage: https://github.com/terms4fairskills/FAIRterminology
      products:
        - id: t4fs.owl
      tracker: https://github.com/terms4fairskills/FAIRterminology/issues
      license:
        url: http://creativecommons.org/licenses/by/4.0/
        label: CC-BY
      usages:
        - user: http://t4fs.esciencedatafactory.com/
          description: Semaphora integrates terms4FAIRskills, allowing users to annotate training materials with the ontology.
        - user: https://fairassist.org
          description: FAIRassist is designed to offer personalised guidance to all stakeholders to enable the discovery of standards and repositories in FAIRsharing, which should be used to make data FAIR, as well as to signpost FAIR assessment resources.
        - user: https://www.fairsfair.eu/
          description: FAIRsFAIR Competence Centre (project wp6) will provide a platform for training materials resulting from project training activities, annotated with the terms4FAIRskills terminology for simple finding and comparison. 
```

It is my feeling that you can use the OBO version IRI for the sake of this submission and the dashboard, so feel free to add it :) Although, your observation is very correct, and I put it on the agenda for today's OBO call.

for the other errors by robot report, see here for examples on how to fix:
https://github.com/INCATools/ontology-development-kit/blob/master/docs/License.md

**Comment by allysonlister** [comment]
Thanks - I'll continue with the updates, that's really helpful :-)

**Comment by allysonlister** [comment]
Good morning all! As you can see from the referenced commit, we have

1. Added ontology versionIRI
2. Updated term IRIs to reflect OBO Foundry best practices http://www.obofoundry.org/id-policy
  - ontology IRI becomes http://purl.obolibrary.org/obo/t4fs.owl
  - term IRIs are in the style http://purl.obolibrary.org/obo/T4FS_0000012
3. Changed dcterms:description to dc:description
4. Added dc:title
5. updated the directory structure 

How do we check this branch against the dashboard to see if all checks now pass? The updated OWL file in the branch is:
 https://github.com/terms4fairskills/FAIRterminology/raw/obo-foundry-updates/t4fs.owl

Thanks!

**Comment by matentzn** [comment]
Hey @allysonlister 

Thats great! Unfortunately, now that you are standardising, the QC will also catch more issues! Basically, because the Dashboard checks only terms in the right namespace, we didn't yet look at your terms. Now we do, here is the report: https://github.com/OBOFoundry/obo-nor.github.io/blob/master/dashboard/t4fs/robot_report.tsv

99% of the problems from the check are very easy fixes, so I hope you are not discouraged.

duplicate_definition (34)
multiple_definitions (26)
label_whitespace (39)
label_formatting (1)

For an explanation of these errors check: http://robot.obolibrary.org/report_queries/

**Comment by allysonlister** [comment]
Thanks - I'm not discouraged :-) We'll take a look and get back to you!

**Comment by matentzn** [comment]
Hey @allysonlister - any updates from your side?

**Comment by allysonlister** [comment]
Thanks for the ping!

We are actively working on the next update, thank you. I hope to submit the next revision of the ontology soon. 

By the way, looking again at Alan's comment, I noticed that GECKO (https://github.com/IHCC-cohorts/GECKO) also maintains a version of the ontology both with an OBO structure (using BFO) and their own tailored view. I think if we restructure our build process to work in a similar way, that should allow us both to have a BFO-aligned version and another version more appropriate for consumption by our user base.

**Comment by allysonlister** [comment]
Apologies for the delay in our replies and for this submission, we have all been very busy in the first half of this year. However, we've completed the portion of the work that would cause the dashboard issues and would like to re-run it using https://github.com/terms4fairskills/FAIRterminology/raw/obo-foundry-updates/development/t4fs.owl

How can I prompt a re-run of the dashboard?

**Comment by matentzn** [comment]
@allysonlister looks great now, congrats:

https://obofoundry.github.io/obo-nor.github.io/dashboard/t4fs/dashboard.html

I took the liberty of updating your metadata to make it  OBO standard conformant:
https://github.com/OBOFoundry/obo-nor.github.io/blob/master/dashboard-config.yml#L191

Please use that metadata moving forward :)

We will assign a human reviewer shortly.

**Comment by allysonlister** [comment]
Thanks! I've fixed the final 4 duplicate definitions and 11 whitespace issues, and committed. I'll leave the few empty definitions just for now, as I'm working on intergrating BFO as Alan suggested via the GECKO build model as mentioned above. Work for that build progresses in the obo-foundry-updates branch (https://github.com/terms4fairskills/FAIRterminology/tree/obo-foundry-updates).

**Comment by matentzn** [comment]
Let us know when you have merged the final version into the `main` branch so we know which file to review!

**Comment by allysonlister** [comment]
I was hesitant to merge against `main` because I didn't want people to be using the OBO IRIs if we weren't going to be accepted ;-) However, if you wish to review against the `main` branch only, then let me know and I'll merge it now. The build procedure work I'm doing now will ultimately add BFO to the top of the hierarchy, but won't otherwise affect the hierarchy.

Would you like to wait until BFO is aligned, or can you get started as is?

**Comment by matentzn** [comment]
I dont mind, you just tell me which exact file I should cycle through dashboard, and we will instruct the reviewers to look at that!

**Comment by allysonlister** [comment]
Thanks, that's very generous. 

https://github.com/terms4fairskills/FAIRterminology/raw/obo-foundry-updates/development/t4fs.owl

Has all of the recent updates, except the BFO upper-level hierarchy (that I'm still working on).

As long as they understand that the BFO part is coming (but not yet there), and are happy to look at the actual T4FS part of the hierarchy, that would be lovely, otherwise let me know.

We intend to release two views going forward: the OBO view with an ULO, and a "community" view that just has the T4FS terms of interest.

**Comment by matentzn** [comment]
https://obofoundry.org/obo-nor.github.io/dashboard/index.html Still some missing metadata in the release, see here for help:

https://oboacademy.github.io/obook/reference/formatting-license/

**Comment by allysonlister** [comment]
Ah, we had the rdfs:comment with the licence information but not the dcterms... Both are present now, at https://github.com/terms4fairskills/FAIRterminology/raw/obo-foundry-updates/development/t4fs.owl

Thank you!

**Comment by allysonlister** [comment]
Good tip! I've added description and title, and committed the changes. Have a great weekend! Thank you for all of your hard work too :-)

**Comment by matentzn** [comment]
All good now, perfect. The board will update in the next 10 minutes, your ontology is doing great.

**Comment by cmungall** [comment]
Not sure if the following are checked by the board

## roots

There are 6 is-a roots - is this intentional?

obo:T4FS_0000372 ! Data stewardship technical concept
obo:T4FS_0000513 ! Role
obo:T4FS_0000519 ! Learning medium
obo:T4FS_0000012 ! Data stewardship activity
obo:T4FS_0000548 ! Data stewardship soft skill
obo:T4FS_0000446 ! Data stewardship guideline

Do you intent to place these under existing COB classes, or propose new COB classes?

(I think this answers your question about BFO)

## naming conventions

All terms are capitalized. This is not prohibited (HP and FMA do this). But it does create stylistic problems for interoperability

## use of OWL

Am I correct in interpreting that you are using existential axioms just to loosely link concepts rather than state actual logical axioms?

For example, Principal Investigator is a subclass of:

'has/wants knowledge about' some 'Data stewardship technical concept'
'has aptitude for' some 'Data stewardship soft skill'
'has/wants practical skill about' some 'Data stewardship technical concept'
'has/wants knowledge about' some 'Data stewardship guideline'
'has/wants competency in' some 'Data stewardship activity'

The existence of a single PI that does has/want any one of these things would yield the ontology inconsistent. I suspect there may be more than one such PI in our domain of discourse.

I don't think there is an OBO principle that talks about matching intent and models of OWL axioms and there are likely other ontologies that do similar things, but it does place t4fs in a slightly different camp than most ontologies in OBO.

## reuse

I see a lot of overlap; e.g. roles/CRO, various information entities/IAO

There are many terms that would presumably be eventually housed in a more general ontology - e.g. everything under Information security

It looks like this is intended as a standalone ontology that doesn't integrate with the rest of OBO - strictly speaking this does not prohibit entry and there are many existing ontologies that are the same. But I wanted to check if you truly wanted to submit this to OBO? If the goal is to make it findable and accessible (it looks v useful as a vocabulary for FAIR evaluation) then this can be done without an OBO submission - eg publish on ontobee, ols, bioportal. Just checking what the overall intent and trajectory for this ontology is, this doesn't constitute to block entry.

**Comment by Drosophilic** [comment]
Thanks for your comments @cmungall. We're aligning T4FS with BFO and IAO where possible,, as there have been some issues integrating with COB - but we're happy to investigate this in the future.

We will make terms all lowercase.

We plan to continue to refine T4FS to react to new and existing use cases. Part of that development will be to reduce overlap with other ontologies. We wish to submit T4FS to OBO in order to improve the consistency and interoperability within T4FS to improve its FAIRness and reusability.

**Comment by matentzn** [comment]
@Drosophilic During yesterday's OBO review call concerns were echoed by other OBO operations members about the issues @cmungall raised:

- "The Use of OWL". What are your plans to address the fact that you are using existential restriction in OWL (all-some logic, i.e. _all_ PIs want _some_ Data stewardship soft skill), when your intention seems to be more to document "some-some", or "SHOULD" logic (all PIs should have Data stewardship soft skill, or _some_ PIs have _some_ Data Stewardship knowledge)
- Would you consider replacing terms that can conceptually be adopted by CRO and IAO prior to admission into OBO to demonstrate the intention to interlink with OBO (which is the _primary_ reason for the existence of the OBO foundry - facilitating an ontology environment of mutually interlinking ontology "modules"). It is very important to understand that _no term that means the same thing in another OBO ontology may be created with a new IRI_.

Thank you for your clarifications! Your case is a bit difficult, unfortunately, compared to our usual submissions like "anatomy" or "disease" ontologies for which we have very clear review criteria - apologies for the piecemeal feedback.

**Comment by nlharris** [comment]
Should this be left open for the ontology owners to reply?

**Comment by matentzn** [comment]
Yeah lets leave it open at least till 18 August (3 months) to give the submitters additional time to respond

**Comment by allysonlister** [comment]
We are working at the moment on finishing the integration with BFO/IAO where appropriate to align with an upper level ontology. And yes, I would be very happy to integrate with others such as CRO where there is overlap - yes, indeed, the orthogonality of OBO ontologies is one of its defining features.

Do you have any particular areas of CRO we could look at? I'm familiar with BFO/IAO but haven't used CRO before.

**Comment by nlharris** [comment]
Can anyone point Allyson to specific areas of CRO to look at?

**Comment by allysonlister** [comment]
> Not sure if the following are checked by the board
> 
> ## roots
> There are 6 is-a roots - is this intentional?
> 
> obo:T4FS_0000372 ! Data stewardship technical concept obo:T4FS_0000513 ! Role obo:T4FS_0000519 ! Learning medium obo:T4FS_0000012 ! Data stewardship activity obo:T4FS_0000548 ! Data stewardship soft skill obo:T4FS_0000446 ! Data stewardship guideline
> 
> Do you intent to place these under existing COB classes, or propose new COB classes?
> 
> (I think this answers your question about BFO)

As @Drosophilic said, we have had some issues with COB - I think we may integrate with BFO, IAO etc separately and then perhaps progressively move towards COB. Our new build procedure is built off of GECKO and the work @jamesaoverton has done there, and I've been having a few conversations with him. I couldn't get COB to work, but I've integrated BFO and am now working on the particular alignment of these 6 classes. You can see ongoing work at https://github.com/terms4fairskills/FAIRterminology/tree/obo-foundry-updates

I'll complete my first pass of alignment with BFO/IAO and then ask for your opinions.

> 
> ## naming conventions
> All terms are capitalized. This is not prohibited (HP and FMA do this). But it does create stylistic problems for interoperability

The branch listed above has all lowercase labels now.

> 
> ## use of OWL
> Am I correct in interpreting that you are using existential axioms just to loosely link concepts rather than state actual logical axioms?
> 
> For example, Principal Investigator is a subclass of:
> 
> 'has/wants knowledge about' some 'Data stewardship technical concept' 'has aptitude for' some 'Data stewardship soft skill' 'has/wants practical skill about' some 'Data stewardship technical concept' 'has/wants knowledge about' some 'Data stewardship guideline' 'has/wants competency in' some 'Data stewardship activity'
> 
> The existence of a single PI that does has/want any one of these things would yield the ontology inconsistent. I suspect there may be more than one such PI in our domain of discourse.
> 
> I don't think there is an OBO principle that talks about matching intent and models of OWL axioms and there are likely other ontologies that do similar things, but it does place t4fs in a slightly different camp than most ontologies in OBO.

You are correct that we are really using these axioms to loosely link concepts. Such axioms came from our community helping us define a model for the ontology as shown here: https://terms4fairskills.github.io/ . We are aiming replicate this accurately, but without too much strictness, in the ontology. Happy to have suggestions for improvements here.

> 
> ## reuse
> I see a lot of overlap; e.g. roles/CRO, various information entities/IAO
> 
> There are many terms that would presumably be eventually housed in a more general ontology - e.g. everything under Information security
> 
> It looks like this is intended as a standalone ontology that doesn't integrate with the rest of OBO - strictly speaking this does not prohibit entry and there are many existing ontologies that are the same. But I wanted to check if you truly wanted to submit this to OBO? If the goal is to make it findable and accessible (it looks v useful as a vocabulary for FAIR evaluation) then this can be done without an OBO submission - eg publish on ontobee, ols, bioportal. Just checking what the overall intent and trajectory for this ontology is, this doesn't constitute to block entry.

This ontology was generated via a number of workshops with community members where we crowdsourced both our model and the initial termset. Our goal is to refactor/deprecate T4FS IRIs where possible as we progressively incorporate other ontologies. 

As you can see from our definitions and definition sources, we have worked closely with other efforts such as CASRAI, which don't have IRIs per se. We want to align with such efforts, and are hampered by this. For example, https://casrai.org/term/principal-investigator no longer exists and we now will need to do a special update that changes our URLs to e.g. https://codata.org/rdm-terminology/principal-investigator/ but this is also in a state of flux. So, for some things, we may need to retain the T4FS IRIs.

I'll comment separately about how to place our top-level terms within ULOs.

**Comment by allysonlister** [comment]
@alanruttenberg said:

> Looks like nice work. Were you thinking of aligning to BFO / other
> ontologies? Some observations: There are many BFO:processes, a number of
> terms that are subs or possibly equivalent to OBI:data transformation and
> IAO:Information content entity. Capabilities are usually considered
> realizable entities - there have been some efforts to formalize them by
> Barry and friends.  The ISO documents would be considered instances of ICEs.

Based on what Alan, Chris and others have said, I proposed the following high-level alignment between t4fs and OBO Foundry:

| t4fs top-level term | New OBO Foundry Parent Class | notes|
|--------------------------|---------------------------------------------|--------|
|role|http://purl.obolibrary.org/obo/BFO_0000023 (role)|I looked at CRO, and OBO Foundry ontology based on CODATA's Credit Taxonomy. However, as shown in the [hierarchy](https://www.ebi.ac.uk/ols/ontologies/cro), role is a top-level term for them. Therefore I have gone for [BFO **role**](https://www.ebi.ac.uk/ols/ontologies/bfo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FBFO_0000023). If you agree, I will refactor our role IRI to instead be the BFO IRI.|
|data stewardship activity| http://purl.obolibrary.org/obo/BFO_0000015 (process) or http://purl.obolibrary.org/obo/OBI_0000011 (OBI planned process)|This hierarchy contains concepts such as "curation" and "archiving". the [OBI planned process](https://www.ebi.ac.uk/ols/ontologies/obi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FOBI_0000011) has a few editor notes which makes me wonder if it's the right class to use, so I've also suggested the BFO process. Which do you think is best?|
|data stewardship guideline|http://purl.obolibrary.org/obo/IAO_0000030 (ICE)| I think this is definitely an [ICE](https://www.ebi.ac.uk/ols/ontologies/obi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FIAO_0000030&lang=en&viewMode=All&siblings=false), but is it something more specific? At the moment, this hierarchy only describes the various components of the FAIR principles, and as such might have something in common with policies or protocols, but where do you really put e.g. guidelines or recommendations? Would [directive information entity](https://www.ebi.ac.uk/ols/ontologies/obi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FIAO_0000033&lang=en&viewMode=All&siblings=false) be better here, as guidelines do tell you what they recommend that you do?|
|data stewardship soft skill|http://purl.obolibrary.org/obo/BFO_0000017 (realizable entity)|Creativity and 'intellectual leadership' are some of the examples of this hierarchy, and may be what Alan was talking about above when he mentioned Capabilities. Therefore I have put this under [realizable entity](https://www.ebi.ac.uk/ols/ontologies/obi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FBFO_0000017), but happy to have some further refinement here.|
|data stewardship technical concept|http://purl.obolibrary.org/obo/IAO_0000030 (ICE)| This is quite a varied hierarchy, so a high-level ICE parent term might be best here.|
|learning medium|http://purl.obolibrary.org/obo/BFO_0000004 (independent continuant)| The items here (book or presentation) are independent continuants (unless I'm missing something), but should I go more finely grained here? While 'book' is a material entity, is a powerpoint presentation? I can't recall, as it's been a while since I've worked properly with BFO.|

Any help with my choices, or suggestions for better ones, are most welcome!

**Comment by cmungall** [comment]
Thanks @allysonlister all my comments have been addressed, we should proceed with registering your ontology.

I suggest we move the discussion as to high level modeling to an issue in your tracker, and link back here for those who are interested. I encourage making issues on repos like COB and IAO if you have issues. If you have questions about the OBI planned process make an issue there (I think many ontologies like IAO and OBI have a lot of "editor notes" from 15 years ago that should be archived and moved somewhere else to avoid confusion)

**Comment by allysonlister** [comment]
Thank you @cmungall and everyone! That's great news - I'll separate out the high-level modelling into our tracker as you suggest, and make appropriate additional tickets as required.

As I'm already updating SWO with the ODK I may do so here too, though I'll leave that as a separate branch and ticket - for now, I'll integrate this branch into the master.

Thanks again!

**Comment by nlharris** [comment]
Congrats, @allysonlister! Can this ticket be closed, or should we leave it open?

**Comment by allysonlister** [comment]
It's up to you - I don't know how these things work? 

I'm going to try to integrate our build procedure for t4fs with the ODK this week, then will merge against our master branch. That means I think t4fs will be ready next week, unless there are unforeseen issues. Is this normally when this ticket gets closed?

Thanks!

**Comment by matentzn** [comment]
The issue will be closed when the metadata is merged into OBO foundry. @allysonlister please ignore my comment here, this is just OBO internal process. @nlharris technically Chris would have to follow the next step in the SOP, which is sending the acceptance message, but if you want you can do that as well: https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/docs/SOP.md#ontology-acceptance-email

**Comment by nlharris** [comment]
I can do that.Do we need an issue tag for "New ontology -- approved"?

**Comment by nlharris** [comment]
Hi @allysonlister, sorry for the delay in finishing the ontology acceptance process!

Thank you again for your ontology submission to the OBO Foundry. We are happy to inform you that your ontology (t4fs) has been accepted following discussion in the OBO Operations Committee meeting, 2022-09-20. Before we can add it to the OBO ontology registry you need to complete the following steps.

Create a metadata record for your ontology to be included in the registry:
1. Create a new file in [https://github.com/OBOFoundry/OBOFoundry.github.io/tree/master/ontology](https://github.com/OBOFoundry/OBOFoundry.github.io/tree/master/ontology), called YOURID.md (there is an “Add file” button in the top right).
2. Obtain the already curated metadata that relates to your ontology from [https://github.com/OBOFoundry/obo-nor.github.io/blob/master/dashboard-config.yml](https://github.com/OBOFoundry/obo-nor.github.io/blob/master/dashboard-config.yml) (you'll need to scroll down to locate yours; it will have your prefix in the " - id:" field.) 
3. Create a pull request to add the metadata record. This pull request should include a link to this issue (the New Ontology Request issue).

Here is an example record for the PATO ontology: [https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/ontology/pato.md?plain=1](https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/ontology/pato.md?plain=1)

Your metadata will be reviewed and merged by a member of the OBO Foundry Operations Committee. Permissible content for fields is being documented [here](https://obofoundry.org/faq/permissible-metadata-content.html).

To create a PURL registry entry for your ontology:
1. Go to [https://github.com/OBOFoundry/purl.obolibrary.org/tree/master/config](https://github.com/OBOFoundry/purl.obolibrary.org/tree/master/config), click “Add file” and add a file named YOURID.yml.
2. Add the desired configuration.
3. Make a pull request with a link to this issue (#1520).

You can let us know in this ticket when that's all done, and then we can close this ticket! Congrats again on being accepted to the OBO Foundry!

**Comment by allysonlister** [comment]
Thanks @nlharris - I'm just making the file now - please can you let me know what 

```
review:
  date: 2010
```

means within the PATO example and how I should use it for my file?

Thanks!

**Comment by matentzn** [comment]
@allysonlister ignore this - you should just take the metadata from the OBO NOR dashboard in (2) as it is!

**Comment by allysonlister** [comment]
Thanks!

BTW can I put two people in the contact field?

**Comment by matentzn** [comment]
no unfortuntely not. Long political discussion

**Comment by allysonlister** [comment]
Ok thanks. One final question.

I've tried to put in a few more of the "products" that the ODK built

```
products:
    - id: t4fs.owl
    - id: t4fs.obo
    - id: t4fs.json
    - id: t4fs-community.owl
    - id: t4fs-community.obo
    - id: t4fs-community.json
```

But how will this know how to build the links if it just has a mirror_from value that points to a particular part of our repo?

```
mirror_from: "https://github.com/terms4fairskills/FAIRterminology/raw/master/t4fs.owl"
```

Thanks!

**Comment by allysonlister** [comment]
Ah, I see, that goes in the yaml file I'm building now...

**Comment by allysonlister** [comment]
OK I think I've done as requested now, but please note that this is my first time making these from scratch, so please suggest any changes you think I should make. Thank you!

**Comment by allysonlister** [comment]
BTW - once this is all completed, how should I approach OLS integration? Does it happen "magically" for all OBO Foundry ontologies, or do I need to notify the OLS people in some way? Thanks

**Comment by matentzn** [comment]
Once all the PRs are merged, you could inform OLS on their issue tracker that its published, and you would like to see it..

**Comment by nlharris** [comment]
Congrats, @allysonlister, and welcome to the OBO Foundry!

Now can this issue be closed?
